virtual/integration/main_test.go

```diff
@@ -197,6 +196,9 @@ func NewServer(
 
 	wmLogger := watermill.LoggerAdapter(watermill.NopLogger{})
 
+	if !flag.Parsed() {
```

Comment:
maybe move this into MainTest (I guess it should be there)

Reply:
you're right, https://golang.org/pkg/testing/#hdr-Main describes that

Reply:
fixed